### PR TITLE
Allows pAIs to be emagged

### PR DESCRIPTION
**Pull request by swissloaf**
Original PR: https://github.com/tgstation/tgstation/pull/57401

# General Documentation

### Intent of your Pull Request

Allows pAI to be emagged, which removes their original master, and allowing a new master to be placed.

### Why is this change good for the game?

This should have been something that happened a long time ago really. Although minor, it enhances the ability of the emag and gives more opportunities for pAIs.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Allows pAI to be emagged, which removes their original master, and allowing a new master to be placed.

### What should players be aware of when it comes to the changes your PR is implementing? 
It allows pAIs to be emagged. God I hate these sections its so redundant.

### What general grouping does this PR fall under? 
Antag weapons changes/updates

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
For example, "This new chem will heal X brute damage". 

# Changelog

:cl:  
tweak: Allows pAIs to be emagged now, removing their original master
/:cl:

### CODE REVIEW

**Comment by ChesterTheCheesy** [comment]
Does this thing improve RP?

**Comment by Moggram** [comment]
[PAI] states "SWISSLOAF HAS A EMAG!"

Maybe I missed it in the code, but I don't see anything stopping a PAI from selling you out after you emag them. Are you going to add to this in other PR or something? Don't get me wrong, its a cool ideal. I just don't know why any one would free PAI if it can still yell about it.

**Comment by swissloaf** [comment]
> [PAI] states "SWISSLOAF HAS A EMAG!"
> 
> Maybe I missed it in the code, but I don't see anything stopping a PAI from selling you out after you emag them. Are you going to add to this in other PR or something? Don't get me wrong, its a cool ideal. I just don't know why any one would free PAI if it can still yell about it.

If they pair the pAI to them, then the pAI is breaking the rules by selling them out
